modules/service/query-service.js

```diff
@@ -44,27 +44,25 @@ class QueryService {
         } else {
             response = await this.dataService.searchByIds(ids, {issuers, types, limit});
         }
+
         return { response, handlerId };
     }
 
     async handleSearchResult(request) {
         // TODO: add mutex
         const { handlerId, response } = request;
-        const handler = await Models.handler_ids.findOne({
-            where: {
-                handler_id: handlerId,
-            },
-        });
-
-        let handlerData = JSON.parse(handler.data);
 
+        const documentPath = this.fileService.getHandlerIdDocumentPath(handlerId);
+        const handlerData = await this.fileService.loadJsonFromFile(documentPath);
+        console.log(response);
```

Comment:
Remove console log from query service

Reply:
Fixed in [this commit](https://github.com/OriginTrail/ot-node/pull/1700/commits/fa27dad009ceb71d8b7cf5c5a54f5feea2408a58).